lib/features/popup-menu/PopupMenuComponent.js

```diff
@@ -232,7 +234,8 @@ export default function PopupMenuComponent(props) {
             entries=${ entries }
             selectedEntry=${ selectedEntry }
             setSelectedEntry=${ setSelectedEntry }
-            onClick=${ onSelect }
+            onSelect=${ onSelect }
+            draggable=${ draggable }
```

Comment:
Where does this magic "draggable" come from? Do we deduce "element is draggable" from a specific popup menu configuration? Or from a `dragstart` handler being provided with the entry?

Reply:
And what is the up or downside of either approach?

(We could likely take inspiration from the context pad and implement exactly what we got there, it served us well).

Reply:
My suggestion would be: Compute the `draggable` property from the presence of the `dragstart` handler in an entries `action` member.

It does not make sense to drag an entry with 

```
{
  label: 'FOO',
  action: (event, ...) => { ... }
}
```

You have to have it `draggable` with the following configuration though: 

```
{
  label: 'FOO',
  action: {
    dragstart: (event, ...) => { ... }
  }
}
```

Reply:
Feels odd to need to pass it on (explicitly).